spec/change_set_persisters/plum_change_set_persister_spec.rb

```diff
@@ -859,4 +859,32 @@
       end
     end
   end
+
+  context 'when persisting a bag' do
+    let(:bag_path) { Rails.root.join('spec', 'fixtures', 'bags', 'valid_bag') }
+    let(:resource) { FactoryBot.build(:archival_media_collection) }
+    let(:change_set_class) { ArchivalMediaCollectionChangeSet }
+    let(:change_set) { change_set_class.new(resource, bag_path: bag_path) }
+
+    before do
+      stub_pulfa(pulfa_id: 'C0652')
+      change_set.prepopulate!
+      change_set.validate(source_metadata_identifier: 'C0652')
```

Comment:
it should create the collection if you didn't provide it; do you need this setup?

Reply:
For the `PlumChangeSetPersister`, I believe so, I've found the following when I attempt to pass an `ArchivalMediaCollectionChangeSet` without an instance of `ArchivalMediaCollection`:
```
Failure/Error: let(:change_set) { change_set_class.new(bag_path: bag_path) }
     
     NoMethodError:
       undefined method `state' for #<Hash:0x00007fc7c5e7b8e8>
```

Similarly, if I don't provide a `source_metadata_identifier` and do not stub a PULFA response:
```
Failure/Error: collection_component: post_save_resource.source_metadata_identifier.first,
     
     NoMethodError:
       undefined method `first' for nil:NilClass
     # ./app/change_set_persisters/plum_change_set_persister/ingest_bag.rb:18:in `run'
     # ./app/change_set_persisters/plum_change_set_persister.rb:171:in `block (2 levels) in after_save_commit'
```

But I have added an additional test for `IngestArchivalMediaBagJob` which shouldn't require that an instance of `ArchivalMediaCollection` have been created.

Reply:
I see; didn't realize we were in the persister spec.